deployment/local/README.md

```diff
@@ -8,53 +8,71 @@ Before running the application locally, ensure the following dependencies are in
 
 ### Required Installations
 
-1. **Docker** - LocalStack runs inside a Docker container.
+1. **Colima/Docker** - LocalStack runs inside a Colima container that uses docker as it's runtime.
 
-   - Install Docker from [Docker's official website](https://www.docker.com/get-started).
+_The install is handled by the run script._
 
-2. **LocalStack** - Provides a local AWS emulating environment.
-
-   - Sign up for a free account: [LocalStack Cloud](https://app.localstack.cloud/sign-up)
-   - Install LocalStack CLI:
-     ```sh
-     brew install localstack/tap/localstack-cli
-     ```
-
-3. **AWS CLI Local** - Required for interacting with LocalStack.
+Links for the curious:
 
-   ```sh
-   pip install awscli-local
-   ```
+- Docker - https://www.docker.com/get-started
+- Colima - https://github.com/abiosoft/colima
 
-4. **AWS CDK Local**
-   ```sh
-   npm install -g aws-cdk-local aws-cdk
-   ```
+Now add this line to the bottom of your bash/zsh rc/profile:
+This is probably the file: `~/.zprofile` if you're using the standard mac setup.
 
-## Open Docker
+```sh
+# this tells docker that you're using colima and to look at colima to answer questions like: are any containers running?
+export DOCKER_HOST="unix://${HOME}/.colima/default/docker.sock"
```

Comment:
I don't have this and I've been fine.

Reply:
So one thing to check is if you are really using colima. Basically start up localstack and then look at docker (`docker ps`). If you see a container, you're not actually using colima.

Reply:
I don't know if that's true.  Look at the demo at the top of their readme at https://github.com/abiosoft/colima

They wouldn't use docker ps to demo if it wasn't their tool behind the scenes.

Reply:
Ok this is great news. So we'll just use the brew install docker version and not have license issues and I'll remove this whole bit.